### ISSUE TITLE
Inter-daemon connection error

### ISSUE THREAD

**Issue by mohierf** [Alignak-monitoring/alignak#767]
Many:
```
[2017-03-28 08:12:35 CEST] WARNING: [alignak.daemons.brokerdaemon] Connection problem to the receiver receiver-master: Request error to http://127.0.0.1:7773/get_running_id: HTTPConnectionPool(host='127.0.0.1', port=7773): Read timed out. (read timeout=3)
[2017-03-28 08:12:40 CEST] WARNING: [alignak.daemons.brokerdaemon] Connection problem to the receiver receiver-master: Request error to http://127.0.0.1:7773/get_running_id: HTTPConnectionPool(host='127.0.0.1', port=7773): Read timed out. (read timeout=3)
```

whereas the receiver master daemon is running and listening:
```
 # ps -aux | grep receiver
root 45909 55.4  0.3 186636  46496 25  I+J   8:21AM   8:42.80 python: alignak-receiver (python2.7)
root 45954 42.7  0.3 238276  51624 25  I+J   8:21AM   6:37.55 python: alignak-receiver-master module: web-services (python2.7)
root 45767  0.0  0.0  25788   3356  -  IsJ   8:21AM   0:00.00 screen -d -S alignak_north_receiver -m sh -c alignak-receiver -c /usr/local/etc/alignak/daemons/North//receive
root 45855  0.0  0.0  25788   3432  -  SsJ   8:21AM   0:00.10 screen -d -S alignak_receiver -m sh -c alignak-receiver -c /usr/local/etc/alignak/daemons/receiverd.ini
root 45771  0.0  0.3 211876  48832 14  Is+J  8:21AM   0:03.46 python: alignak-receiver receiver-north (python2.7)
root 45901  0.0  0.3 182284  46332 14  S+J   8:21AM   0:00.06 python: alignak-receiver (python2.7)
root 45950  0.0  0.3 211868  47928 14  S+J   8:21AM   0:00.03 python: alignak-receiver-north module: nsca_north (python2.7)
root 45857  0.0  0.3 215220  54296 25  Ss+J  8:21AM   0:09.34 python: alignak-receiver receiver-master (python2.7)
root 45952  0.0  0.3 212132  48964 25  S+J   8:21AM   0:00.03 python: alignak-receiver-master module: nsca (python2.7)
root 48856  0.0  0.0  18832   1956  0  S+J   8:38AM   0:00.00 grep receiver

```

**Note** that despite the periodical error log, everything is running fine!

**Comment by mohierf** [comment]
Same problems between the broker and the poller:
```
[2017-03-28 11:12:16 CEST] WARNING: [alignak.daemons.brokerdaemon] Connection problem to the poller poller-north: Request error to http://127.0.0.1:17771/get_running_id: HTTPConnectionPool(host='127.0.0.1', port=17771): Read timed out. (read timeout=3)
[2017-03-28 11:12:19 CEST] WARNING: [alignak.daemons.brokerdaemon] Connection problem to the poller poller-north: Request error to http://127.0.0.1:17771/get_running_id: HTTPConnectionPool(host='127.0.0.1', port=17771): Read timed out. (read timeout=3)
```

**Note** that the raised exception is simply a time out notification. Perharps that waiting only 3 seconds for a daemon response is a bit too short ?

@ddurieux @Seb-Solon what do you think about this?

**Comment by Seb-Solon** [comment]
timeout is a parameter that can be changed as far as I remember. For distant satellite that would make sense to increase it.

**Comment by mohierf** [comment]
Sure it is a parameter. There is a `ping` timeout (3 seconds) and a `data` timeout (10 seconds). Note that in my configuration all the satellites are not remote but located on the same server ...

**Comment by ddurieux** [comment]
3 seconds short? I think there is another problem, because in local not have an answer in this 3 seconds not normal, I prefer search the cause of the problem instead try increase the timeout time ;)

**Comment by mohierf** [comment]
This problem was fixed thanks to #769
